api/src/pcapi/core/offers/api.py

```diff
@@ -894,21 +896,28 @@ def add_criteria_to_offers(
     return True
 
 
-def reject_inappropriate_products(ean: str) -> bool:
+def reject_inappropriate_products(ean: str, **kwargs: typing.Any) -> bool:
+    skip_send_booking_cancellation_emails: bool = kwargs.get("skip_send_booking_cancellation_emails") is True
```

Comment:
(style) Il est dangereux d'avoir des arguments non nommés comme ça. Un.e collègue pourrait appeler la fonction avec `skip_send_booking_cancelation_emails=True` et personne (pas même le code !) ne verrait la typo. Ici, on sait ce qu'on attend, donc il est préférable de nommer l'argument. Cela permet aussi de voir, **dans la signature de la fonction**, la valeur par défaut de l'argument et donc le **comportement** par défaut de la fonction.

```suggestion
def reject_inappropriate_products(ean: str, send_booking_cancellation_emails: bool = True) -> bool:
```

Au passage, je pense qu'on devrait inverser le nommage ("send_..." au lieu de "skip_send...") pour éviter la double négation "skip_send...=False". Il me semble plus simple de lire/écrire "send=True" et "send=False" (plutôt que "skip_send=False" et "skip_send=True").